src/ekka.app.src

```diff
@@ -1,6 +1,6 @@
 {application, ekka,
  [{description, "Autocluster for EMQ X Broker"},
-  {vsn, "0.15.11"},
+  {vsn, "0.15.12"},
```

Comment:
I'd suggest we use git tags to avoid forgetting to update the vsn here.

```suggestion
  {vsn, "git"},
```

Reply:
done